### PR TITLE
Lucas branch

### PR DESCRIPTION
**Pull request by ortengren**

### CODE REVIEW

File: assets/img/Snapchat-1493922749~2.jpg:Zone.Identifier
Comment (cbefan) [inline assets/img/Snapchat-1493922749~2.jpg:Zone.Identifier:1]: I don't think this is necessary? Also windows really does not like the '.jpg:Zone.Identifier' file extension on this.

Comment (ortengren) [reply assets/img/Snapchat-1493922749~2.jpg:Zone.Identifier:1]: Weird, I didn't mean for that to be added. Maybe it was some image metadata. I've deleted it now.

**Comment by cbefan** [comment]
Webpage works and looks great, although there will be some conflicts with merging because of how old the branch is. Github should be able to figure it out.

**Comment by rosecers** [comment]
I would suggest @ortengren rebase then rerun `make_member_page.py`.

Lucas, you can do this by:
```
git fetch --all
git rebase main 
python make_member_page.py
git add members/index.md
git commit -m "rebasing and adding member page"
git push
```

File: members/lucas_ortengren.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+<img src="/website/assets/img/lucas_ortengren.png" alt="Lucas Ortengren Image" class="center" style="max-width: 100%">
+
+<!-- Replace `Example Student` with your name and include your start date-->
+# **Lucas Ortengren (2023 - present)**
+
+<!-- Choose your title -- feel free to be professionally silly -->
+**Role in the Cersonsky Lab**: Documentation-writer and miscellaneous assistant.
+
+<!-- Name at least one research topic amongst this list -->
+**Things I'm excited about right now**: Learning the ins and outs of aniSOAP and machine learning.
```
Comment (rosecers) [inline members/lucas_ortengren.md:15]:
```suggestion
**Things I'm excited about right now**: Learning the ins and outs of AniSOAP and machine learning.
```

**Comment by rosecers** [comment]
@ortengren how is this looking?

**Comment by ortengren** [comment]
@rosecers I fixed the capitalization picture, added some more pictures, and added a little bit to the text of my member page.  It should be good!

**Comment by rosecers** [comment]
Hey @ortengren! I've pushed one change to the CSS to make your photo grid render a bit better. It's good to squash and merge!

**Review by rosecers — APPROVED** [review]

**Pull request merged**